lib/deploy_pin/runner.rb

```diff
@@ -4,8 +4,14 @@
 module DeployPin
   module Runner
     def self.run(identifiers:)
-      DeployPin::Collector.new(identifiers: identifiers).run do |index, count, task, executable|
-        self.print("[#{index + 1}/#{count}] Task #{task.title} #{task.uuid}##{task.group} (#{executable ? 'run' : 'skip'})")
+      DeployPin::Collector.new(identifiers: identifiers).run do |index, count, task, executable, start:|
+        end_of_msg = if executable
+          start ? '(Run)' : "(Done)\n\n"
```

Comment:
let's add a message like this:
`Done in 30 seconds`

Reply:
would be great to create a formatter parameter for the task execution output.
you should be able to pass it as a second argument like ...run(identifiers:, formatter: ->(task attributes) { "the output message #{task_attributes[:execution_time]}" }
also please define default formatter